src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  border: #f3f3f3 1px solid;
+  border-radius: 5px;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.text {
+  height: 36px;
+  color: #060b35;
+  margin: 0 16px;
+  line-height: 18px;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin: 4px 0 0 16px;
+  line-height: 14px;
+  width: 98px;
+}
+
+.stars {
+  display: flex;
+  width: 166px;
+  margin: 16px 0 0 16px;
+}
+
+.stars__stars {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+  background-image: url(/src/images/star.svg);
+}
+
+.stars__stars:first-child {
+  margin-left: 0;
+}
+
+.stars__stars:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.reviews {
+  color: #060b35;
+  display: flex;
+  font-weight: 400;
+  font-size: 10px;
+  flex-direction: row-reverse;
+  align-items: center;
+  margin-left: 17px;
+  width: 53px;
+}
+
+.price {
+  width: 166px;
+  height: 18px;
+  display: flex;
+  white-space: nowrap;
+  margin: 24px 0 0 16px;
+}
+
+.price__text {
+  color: #616070;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price__tag {
+  color: #060b35;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 18px;
+  margin-left: 85px;
+}
+
+.link {
+  display: flex;
+  font-weight: 700;
+  margin: 16px;
+  font-size: 14px;
+  width: 166px;
+  min-height: 40px;
+  background-color: #00acdc;
+  text-decoration: none;
+  border-radius: 5px;
+  color: #fff;
+  align-items: center;
+  justify-content: center;
+}
+
+.link:hover {
+  min-height: 38px;
+  width: 164px;
```

Comment:
The hover effect reduces the size of the `.link` element, which might cause layout shifts. Consider maintaining the same size for the element during hover to avoid such shifts.